database/model.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const { descriptionSchema } = require('./db');
+
+const Description = mongoose.model('Description', descriptionSchema);
+
+module.exports = {
+  getOne: (id) => (
```

Comment:
I never thought to add the database functions as methods of exports.  This is pretty genius!